### PR TITLE
[14.0] [MIG] sale_company_currency

### PR DESCRIPTION
**Pull request by alexeirivera87**

### CODE REVIEW

**Review by ajaniszewska-dev — APPROVED** [review]

**Comment by rousseldenis** [comment]
@alexeirivera87 Thanks for your contribution. Next time can you follow migration guide? Especially 'Full process' part.

https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-14.0#technical-method-to-migrate-a-module-from-130-to-140-branch

File: sale_company_currency/models/sale_order.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        store=True,
+    )
+
+    @api.depends("amount_total", "currency_rate")
+    def _compute_amount_company(self):
+        for order in self:
+            if order.currency_id.id == order.company_id.currency_id.id:
+                to_amount = order.amount_total
+            else:
+                to_amount = order.amount_total / order.currency_rate
```
Comment (yankinmax) [inline sale_company_currency/models/sale_order.py:32]: hello) why don't you use standard conversion odoo method?
https://github.com/odoo/odoo/blob/af23ab272fcf9c33980679b0215be2fff0ea1a86/odoo/addons/base/models/res_currency.py#L241

**Comment by alexeirivera87** [comment]
Hello @rousseldenis 

I updated the PR, I followed the migration guide, including the 'Full process' part. Let me know if everything is OK now.

Regards

**Review by rousseldenis — APPROVED** [review]
Code review

**Pull request merged**